Classes/Issues/Comments/QuotedComment.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  QuotedComment.swift
+//  Freetime
+//
+//  Created by Joan Disho on 09.05.18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+class QuotedComment {
+    private let feed: Feed
+    private let commentModel: IssueCommentModel
+
+    init(in feed: Feed, with commentModel: IssueCommentModel) {
+        self.feed = feed
+        self.commentModel = commentModel
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    var quote: String {
+        let text = commentModel.rawMarkdown.substringUntilNewLine()
+        return ">\(text)\n\n@\(commentModel.details.login)"
+    }
+
+    func scrollInto() {
```

Comment:
`scrollTo`*? Also feel like this shouldn't be the responsibility of this class.

Reply:
I agree with you, it feels like it's not the responsibility of this class to scroll. 
However, since this line of code is repeated , I decided to do it like it is.